app.js

```diff
@@ -26,6 +26,14 @@ if (!process.env.RACKSPACE_USERNAME
   throw new Error('Required parameters not provided from the environment');
 }
 
+routes.client.createContainer({
```

Comment:
Might also be worth a comment here to explain that create no-ops if the container is already present.